docs/guides/bugzilla.rst

```diff
@@ -0,0 +1,192 @@
+Bugzilla Guide
+==============
+
+Contributor guidelines
+----------------------
+We have developed several tools/workflows to make the process
+for dealing with Bugzilla bugs (BZ's) easier for contributors.
+Here, we discuss:
+
+* Metadata markers for test cases: ``blockers``, ``coverage``, and ``automates``
+* the ``miq bz`` command
+* ``Bugzilla:`` docblocks
+
+Metadata Markers
+^^^^^^^^^^^^^^^^
+* **blockers**: this marker is used when a test case is `blocked` by a bugzilla bug.
+  To use this marker simply add a ``@pytest.mark.metadata(blockers=[BZ(<bug_id>])`` decorator
+  to you test case. At run time, the property ``BZ(<bug_id>).blocks`` will be checked,
+  if ``True``, the test case will be skipped. For example,
+
+  .. code-block:: python
+
+     @pytest.mark.meta(blockers=[BZ(1234567)])
+     def test_blocked():
+        """
+        This test case is blocked by the BZ with id 1234567
+        """
+        pass
+
+
+  There are several key word arguments
+  that can be passed to this: ``uncollectif``, ``forced_streams``, etc. Look into the
+  codebase for examples of their usage.
+* **coverage**: this marker is used to denote `manual` test cases that are providing
```

Comment:
I think `manual` has not been formatted properly, it appears like a normal text when viewed [here](https://github.com/ManageIQ/integration_tests/pull/9190/files?short_path=6aacf50#diff-6aacf50c1d0bd2c22063238a140cc7ab).

Reply:
I believe this is a small difference between GH markdown and readthedocs markdown, this will show up as _italics_ in the html file.

Reply:
![Screenshot from 2019-08-14 08-22-27](https://user-images.githubusercontent.com/44065123/63020645-af66b500-be6c-11e9-91ec-701ccc552561.png)